scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 547,
```

Comment:
Does the size of the min not increase too?

Reply:
I think it does
![Screen Shot 2019-10-16 at 11 55 25](https://user-images.githubusercontent.com/25608386/66913164-02dfb580-f00c-11e9-9292-c39650eeea90.png)

Reply:
Good point, thank you